ui/src/components/ResultBlock/index.tsx

```diff
@@ -7,8 +7,9 @@ import { Log, LogIndent } from "basics/Log";
 import { Result, NetworkCall } from "types/testCases";
 
 const ResultBlockWrapperEl = styled.div`
-  border-radius: 0.25rem;
-  border: 1px solid #fbfaf7;
+  border-bottom-left-radius: 0.25rem;
+  border-bottom-right-radius: 0.25rem;
+  border: 1px solid var(--pal-border-primary);
```

Comment:
A small nit: the top of the box is bolder because of the border of the div above it.

One way this could be less noticable is if we didn't include a border-top on the result block,  but it's still slightly bolder. Is there a way to remove the bottom border of the test name div if there is a failure?

Reply:
<img width="1549" alt="Screen Shot 2021-08-03 at 2 35 31 PM" src="https://user-images.githubusercontent.com/6789586/128068164-844c7153-df96-40a4-9473-bf0d21d06f1c.png">

Addressed with latest commit